MAINTAINERS.md

```diff
@@ -0,0 +1,21 @@
+## Overview
+
+This document contains a list of maintainers in this repo.
+See [GOVERNANCE.md](GOVERNANCE.md) that explains the function of this file.
+
+## Current Maintainers
+
+| Maintainer          | GitHub ID                                       | Affiliation |
+| ------------------- | ----------------------------------------------- | ----------- |
+| Zhu Binbin          | [enjoy-binbin](https://github.com/enjoy-binbin) | Tencent     |
+| Wen Hui             | [hwware](https://github.com/hwware)             | Huawei      |
+| Madelyn Olson       | [madolson](https://github.com/madolson)         | Amazon      |
+| Ping Xie            | [pingxie](https://github.com/pingxie)           | Google      |
+| Zhao Zhao           | [soloestoy](https://github.com/soloestoy)       | Alibaba     |
+| Viktor Zuiderkwast  | [zuiderkwast](https://github.com/zuiderkwast)   | Ericsson    |
```

Comment:
Correcting my name.

My github username is a funny spelling (like if it were Dutch, which it isn't).

```suggestion
| Viktor Söderqvist  | [zuiderkwast](https://github.com/zuiderkwast)   | Ericsson    |
```